swift.en.md

```diff
@@ -0,0 +1,340 @@
+# Swift coding style
+
+## Introduction
+
+This document defines the conventions for writing Swift code at Cookpad Inc.
+
+This guide directed for Swift 3.0.2.
+
+## Objective
+
+This guides aim to accomplish these objectives.
+
+1. Clarify purposes 
+1. Improve readability
+1. Keep consistency
+1. Reduce side-effects
+
+## Code style
+
+- **[MUST]** Use four spaces for 1-level of indent.  Do not use the horizontal tab character.
+- **[MUST]** Add linebreak to end of files
+- **[MUST]** Place open brackets on same lines
+
+```swift
+// Bad
+func update() 
+{
+}
+
+// Good
+func update() {
+}
+```
+
+- **[MUST]** Use trailing comma on elements of collections
+    - Because of reducing file diffs
```

Comment:
@halogenandtoast @kittinunf 

> Why is this a must? 

We think this rule helps to reduce unnecessary diff lines when PR and make it easy for reviewers.

There is the linter on SwiftLint.
We enabled this, people don't care about this anymore.

https://github.com/realm/SwiftLint/blob/master/Source/SwiftLintFramework/Rules/TrailingCommaRule.swift

How do you think about?